src/app/globals.css

```diff
@@ -2,20 +2,13 @@
 @tailwind components;
 @tailwind utilities;
 
-:root {
-  --background: #ffffff;
-  --foreground: #171717;
-}
-
-@media (prefers-color-scheme: dark) {
-  :root {
-    --background: #0a0a0a;
-    --foreground: #ededed;
-  }
+@font-face {
```

Comment:
Lets please keep the default fonts for now. We can create separate task for font customization. This is because font changes affect entire application experience.

Reply:
okay sir